db/schema.rb

```diff
@@ -237,6 +243,40 @@
     t.index ["user_id"], name: "IX_organisation_user_user_id"
   end
 
+  create_table "pets2", id: false, force: :cascade do |t|
```

Comment:
'pets2'? 😆

Reply:
I have no idea what sort of foul play is at work here 🤣 I've removed it anyway.